internal/use_cases/create_pull_request.go

```diff
@@ -224,7 +254,32 @@ func (cpr *CreatePullRequest) getPullRequestTitleAndBody(issue domain.Issue) (ti
 		body = fmt.Sprintf("Relates to [%s](%s)", issue.ID(), issue.URL())
 
 	default:
-		err = fmt.Errorf("issue tracker %s is not supported", issue.TrackerType())
+		return "", "", fmt.Errorf("issue tracker %s is not supported", issue.TrackerType())
+	}
+
+	// If template path is provided, read and append it after the issue reference
+	// (we already validated that the file exists in validateTemplateFile)
+	if cpr.Cfg.TemplatePath != "" {
+		var templateContent []byte
+
+		// If not an absolute path, resolve from repository root
+		templatePath := cpr.Cfg.TemplatePath
+		if !filepath.IsAbs(templatePath) {
+			repoRoot, err := cpr.Git.GetRepositoryRoot()
+			if err != nil {
+				return "", "", fmt.Errorf("failed to determine repository root: %w", err)
+			}
+			templatePath = filepath.Join(repoRoot, templatePath)
+		}
+
+		// Read template file (we know it exists because we checked in validateTemplateFile)
+		templateContent, err = os.ReadFile(templatePath)
+		if err != nil {
+			return "", "", fmt.Errorf("failed to read template file: %w", err)
+		}
```

Comment:
suggestion: I would call here the normalize function I talked about in some previous comment to retrieve the absolute path value.